auth/vitest.config.ts

```diff
@@ -1,32 +1,8 @@
-import { defineConfig, coverageConfigDefaults } from "vitest/config"
-import { VitestCucumberPlugin } from "@amiceli/vitest-cucumber"
+import { defineProject, mergeConfig } from "vitest/config"
+// eslint-disable-next-line @nx/enforce-module-boundaries
```

Comment:
NX doesn't like the use of relative paths - it recommends defining paths in the root tsconfig